### DISCUSSION TITLE
和官服能共存吗？|Does it conflict with the official version？

### DISCUSSION THREAD

**Discussion by Waynehhhh** [Melledy/Grasscutter#1461] [Q&A] [answered]
架设好之后还能玩官服吗？就是两个版本能不能同时存在，且互不影响   电脑端
Can I play official clothes after erection? Is whether the two versions can coexist at the same time？by computer.

**Comment by KisaraKira** [comment, marked as answer]
This project is just a server which mean it can't affect your local game client.
That's your launcher's matters,for now it has no conflict with the official game client.
You can also make a copy of your game file to separate them.
